### ISSUE TITLE
Get the current slide ( src ) and use it outside the scope

### ISSUE THREAD

**Issue by lexcaraig** [sguti/ng2-owl-carousel2#1]
Hi, i would like to ask if something like this is possible? I am currently working on this. Any help? I really appreciate it.

```
<div>
    // i would actually like to put the image.preview here
</div>

<ng2-owl-carousel2
     [options]="carouselOptions">
            <div class="owl-item" *ngFor="let image of carousel; let i = index">
                <img [src]="image.cover" alt="image.title">
                 // it has image.preview
            </div>
 </ng2-owl-carousel2>
```

Thanks,
Best Regards, Lex

**Comment by sguti** [comment]
Hi, You want to put the image(as a preview) out of the carousel? Is it for a particular item(selected)? If yes then simply raise an event and sent the image details as payload and display that image.

Anyway this code needs lots of changes. If you need I make those changes.

Thanks,
Sudip

**Comment by lexcaraig** [comment]
@sguti Yes, please. If you can make it. It will be a lot helpful with the others too. Thanks a lot!

**Comment by sguti** [comment]
@lexcaraig checout the updated version. Let me know if you are facing any issue. Go through the readme there are few changes.

**Comment by lexcaraig** [comment]
@sguti Great! I'll check it out. I will close this ticket once I confirm its working on my side. Thanks a lot!

**Comment by lexcaraig** [comment]
@sguti hey man! I checked the repo and saw your last commit 441e323680fe6bd1a245df0e075609240ce5df50 it was on Jun 9? Sorry, am I getting the latest version? Thanks!

**Comment by lexcaraig** [comment]
Oh, my bad. I just checked the https://www.npmjs.com/package/ng2-owl-carousel2 and its updated. I wonder if it will be updated when I install it? Anyway, thanks a lot! Will try this again.

**Comment by lexcaraig** [comment]
And also, can you make a clearer documentation? Its hard for me to follow it. I think its not working on me. Thanks!

**Comment by sguti** [comment]
Sorry to hear that. Are you getting any error or just the carousel is not working?
